app/models/Product.scala

```diff
@@ -1,21 +1,22 @@
 package models
 
+import play.api.db.{Database, NamedDatabase}
 import play.api.libs.json.{JsMacroImpl, Json, OFormat}
 
+import javax.inject.Inject
+import scala.concurrent.Future
 import scala.language.experimental.macros
 
 trait ImplicitJsonFormat
 trait JsonImplicits {
   implicit def implicitJsonFormat[A <: ImplicitJsonFormat]: OFormat[A] = macro JsMacroImpl.implicitConfigFormatImpl[A]
 }
 
-case class Product(service: String, environment: String, serverTime: String)
+case class EndPointStatus(service: String, environment: String, serverTime: String)
 
-/** Object Companion for serialization
- *
- */
-object Product extends JsonImplicits {
-  implicit val format: OFormat[Product] = Json.format
+object EndPointStatus extends JsonImplicits {
+  implicit val format: OFormat[EndPointStatus] = Json.format
 }
 
-case class NestedCaseProduct(value: String) extends ImplicitJsonFormat
+case class Product(ID: String, name: String,quantity: Option[String],price: Option[String],
+                   createdAt: Option[String],updatedAt: Option[String])
```

Comment:
place parameters in a vertical list for readability

Reply:
Will do from now on!